src/main/java/frc/robot/commands/liftUp.java

```diff
@@ -0,0 +1,52 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
Is liftDown Command needed if this one is passing a joystick value? If not remove lift down and rename this to runLift or something.